nodemon.json

```diff
@@ -0,0 +1,9 @@
+{
+  "restartable": "rs",
+  "ignore": [
+    ".git",
+    "node_modules/**/node_modules"
+  ],
+  "verbose": true,
+  "watch": ["./pages", "./src", "./server"]
```

Comment:
I think we only need server side to be watch, why need to include client script?

Reply:
Page, src include component it will need to be render in server-side. So It we need to watch it in server too.